Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -11530,28 +11541,38 @@ void WebPageProxy::focusedFrameChanged(IPC::Connection& connection, std::optiona
     broadcastFocusedFrameToOtherProcesses(connection, WTFMove(frameID));
 }
 
-void WebPageProxy::processDidBecomeUnresponsive()
+void WebPageProxy::processDidBecomeUnresponsive(WebProcessProxy& process)
 {
     WEBPAGEPROXY_RELEASE_LOG_ERROR(Process, "processDidBecomeUnresponsive:");
 
     if (!hasRunningProcess())
         return;
 
+    bool wasResponsive = m_unresponsiveProcesses.isEmpty();
```

Comment:
This is not necessarily true if m_unresponsiveProcesses contained only other processes.

Reply:
? If there are other unresponsive processes, then we weren't responsive?

Reply:
I also don't completely understand the comment. Maybe a concrete example would be useful?